client/QuickStart/API_Util.cs

```diff
@@ -0,0 +1,410 @@
+            }
+
+            if (publicCertFilePath != null)
+            {
+                try
+                {
+                    publicCertificate = File.ReadAllText(publicCertFilePath);
+                }
+                catch (InvalidOperationException e)
+                {
+                    ErrorHandler($"Failed to load public certificate file from {publicCertFilePath}", e);
+                }
+                return new KustoConnectionStringBuilder(clusterUrl).WithAadApplicationSubjectAndIssuerAuthentication(appId, SubjectDistinguishedName, IssuerDistinguishedName, tenantId);
+            }
+
+
+            X509Certificate2 certificate = null;
+            if (certificatePath != null)
+                certificate = new X509Certificate2(certificatePath, certificatePassword);
+            else
+                ErrorHandler("Missing Certificate path!");
+
+            return new KustoConnectionStringBuilder(clusterUrl).WithAadApplicationCertificateAuthentication(applicationId, certificate, tenantId, sendX5c: true);
+        }
+
+        /// <summary>
+        /// Creates a fitting ClientRequestProperties object, to be used when executing control commands or queries.
+        /// </summary>
+        /// <param name="scope">Working scope</param>
+        /// <param name="timeout">Requests default timeout</param>
+        /// <returns>ClientRequestProperties object</returns>
+        public static ClientRequestProperties CreateClientRequestProperties(string scope, string timeout = null)
+        {
+            var clientRequestProperties = new ClientRequestProperties
+            {
+                Application = "QuickStart.csproj",
+                // It is strongly recommended that each request has its own unique request identifier.
+                // This is mandatory for some scenarios (such as cancelling queries) and will make troubleshooting easier in others.
+                ClientRequestId = $"{scope};{Guid.NewGuid().ToString()}"
+            };
+            // Tip: Though uncommon, you can alter the request default command timeout using the below command, e.g. to set the timeout to 10 minutes, use "10m"
+            if (timeout != null)
+                clientRequestProperties.SetOption(ClientRequestProperties.OptionServerTimeout, timeout);
+
+            return clientRequestProperties;
+        }
+
+        /// <summary>
+        /// Executes a Command using a premade client
+        /// </summary>
+        /// <param name="client">Premade client to run Commands. can be either an adminClient or queryClient</param>
+        /// <param name="configDatabaseName">DB name</param>
+        /// <param name="command">The Command to execute</param>
+        /// <returns>True on success, false otherwise</returns>
+        public static async Task ExecuteAsync(IDisposable client, string configDatabaseName, string command)
+        {
+            try
+            {
+                Newtonsoft.Json.Linq.JObject[] result;
```

Comment:
Don't need fully qualified name

Reply:
You need a using. Try to see if the IDE/vscode can do it for you.

Reply:
VSCODE is incapable of such stuff.
anyway, seems like JObject is not IDisposable - so we cant really use a using block here (see here: https://stackoverflow.com/questions/42077388/type-used-in-a-using-statement-must-be-implicitly-convertible-to-system-idispos) 
 
![image](https://user-images.githubusercontent.com/100577889/172853657-0db6a224-89c1-44ad-b22f-d9ad8029a91f.png)

Reply:
I meant `using Newtonsoft.Json.Linq;`, not using statement, we're good.